frontend/public/components/masthead.jsx

```diff
@@ -49,10 +50,20 @@ export const getBrandingDetails = () => {
 
 export const Masthead = ({ onNavToggle }) => {
   const details = getBrandingDetails();
+  const defaultRoute = '/';
+  const logoProps = {
+    href: defaultRoute,
+    // use onClick to prevent browser reload
+    onClick: e => {
+      e.preventDefault();
+      history.push(defaultRoute);
+    },
+  };
 
   return (
     <PageHeader
       logo={<Brand src={details.logoImg} alt={details.logoAlt} />}
```

Comment:
@jeff-phillips-18 It would be better if we could pass

```jsx
logo={<Link to="/"><Brand src={details.logoImg} alt={details.logoAlt} /></Link>}
```

here, but @rhamilto says that patternfly-react *always* add an `a` element even if we don't pass an `href`.This means we'd have an `a` inside an `a`.  So we have to do a hacky workaround to make this work with react-router.

Is there any chance to fix this upstream so we can have the cleaner implementation?

Reply:
Without the fix:  ![screen shot 2019-01-14 at 10 02 09 am](https://user-images.githubusercontent.com/895728/51120634-87827680-17e3-11e9-8a25-525562f03f77.png)

Reply:
@spadgett Did/can you file an issue in patternfly-react?